src/Sparrow.Server/Utils/TaskExecutor.cs

```diff
@@ -116,10 +116,19 @@ public void Execute(object state)
                 if (callback == null)
                     return;
 
-                if(Interlocked.CompareExchange(ref _callback, null, callback) != callback)
+                if (Interlocked.CompareExchange(ref _callback, null, callback) != callback)
                     return;
 
-                callback(state);
+                try
+                {
+                    callback(state);
+                }
+                catch (Exception e)
+                {
+                    var logger = Logging.LoggingSource.Instance.GetLogger<RunOnce>("TaskExecutor");
```

Comment:
Do we want to create new logger instance per each exception?

Reply:
We saw it once in 4 years, I don't think that this is going to be a hot spot